config/locales/idv/fr.yml

```diff
@@ -161,8 +161,8 @@ fr:
       sessions:
         id_information_message: tel qu'il apparaît sur votre carte d'identité officielle
         id_information_subtitle: Informations d'identification
-        no_pii: N'utilisez pas de véritables données personnelles (il s'agit d'une
-          démonstration seulement)
+        no_pii: FAKE N'utilisez pas de véritables données personnelles (il s'agit
+          d'une FAKE démonstration seulement)
```

Comment:
I think the french word we want is TRUQUÉ, and the second instance should come after the closing parenthesis.

Reply:
Done